modules/release-notes/snip-bug-fix-rhidp-1334.adoc

```diff
@@ -0,0 +1,9 @@
+[id="bug-fix-rhidp-1334"]
+= Removed unnecessary pull secret from the {product} Helm Chart
```

Comment:
*sigh* 4 lines that could be one, or none. This is hardly noteworthy. 

I promise to try to stop complaining about the verbosely overwordy garrulous redundant way we document even trivial tech debt updates.